pkg/apis/dashboard/v0alpha1/search.go

```diff
@@ -69,9 +69,9 @@ type DashboardHit struct {
 	// The display nam
 	Title string `json:"title"`
 	// Filter tags
-	Tags []string `json:"tags,omitempty"`
+	Tags []string `json:"tags"`
 	// The UID/name for the folder
-	Folder string `json:"folder,omitempty"`
+	Folder string `json:"location"`
```

Comment:
Any reason to change from folder?

Reply:
the front end wants it to be `location` which is outside of the searcher implementation.

Reply:
i had to map `name` to `title` so i could do it on the front end if you want to keep this as `folder`